frontend/src/components/grantaccess/form/UserNameInput.vue

```diff
@@ -16,6 +18,8 @@ const props = defineProps({
     helperText: { type: String },
 });
 
+const PERMISSION_REQUIRED_FOR_OPERATION = 'permission_required_for_operation';
```

Comment:
could we check if we can import this from 'fam-app-acsctl-api'? if not possible that's okay

Reply:
I did not found it on 'fam-app-acsctl-api', but I'll take another look